components/admin_console/card/card.scss

```diff
@@ -0,0 +1,45 @@
+.Card__card {
+    background-color: var(--sys-center-channel-bg);
+    border: 1px solid rgba(var(--sys-center-channel-color-rgb), 0.08);
+    box-shadow: 0px 2px 3px rgba(0, 0, 0, 0.08);
+    border-radius: 4px;
+    color: var(--sys-center-channel-color);
+    width: 100%;
+    height: '463px';
+    margin-bottom: 20px;
+}
+
+.Card__cardHeader {
```

Comment:
Your naming style seems to follow smurf naming, which is an anti-pattern. https://devcards.io/smurf-naming-convention

We should be able to avoid this if we just override the base styles as I mentioned above.